include/ignition/gui/qml/IgnCard.qml

```diff
@@ -222,12 +242,20 @@ Pane {
 //        height: cardWindowContent.height
 //      }
 //    },
+    // Floating and Docked state are the expanded states
     State {
       name: "docked"
     },
-
     State {
       name: "floating"
+    },
+    // Docked collapsed state
+    State {
+      name: "d_collapsed"
```

Comment:
How about being more verbose for clarity? `docked_collapsed` / `floating_collapsed` would be clearer to people not familiar with the code.